### DISCUSSION TITLE
Easy contributions! Translate docs for MMTracking ~

### DISCUSSION THREAD

**Discussion by JingweiZhang12** [open-mmlab/mmtracking#760] [General]
Hi guys, we are planning to **translate docs** from English to Chinese in MMTracking. PRs from the community are welcomed :stuck_out_tongue_closed_eyes:. To make the community fully aware of the progress, we list the progress here. Please feel free to leave a message and create a PR if you are willing to do these :rocket::rocket:.

## Translate docs
- [ ] docs part-1 @ArtechStark 
   - [ ] [1_config.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/user_guides/1_config.md)
   - [ ] [2_dataset_prepare.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/user_guides/2_dataset_prepare.md)
   - [ ] [3_inference.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/user_guides/3_inference.md)
- [ ] docs part-2 @LihanChen2004 https://github.com/open-mmlab/mmtracking/pull/795
   - [ ] [4_train_test.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/user_guides/4_train_test.md) 
   - [ ] [5_visualization.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/user_guides/5_visualization.md) 
- [ ] docs part-3 @l412198735 https://github.com/open-mmlab/mmtracking/pull/799
   - [ ] [6_analysis_tools.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/user_guides/6_analysis_tools.md)
   - [ ] [get_started.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/get_started.md)
- [ ] docs part-4 @DeemoZen https://github.com/open-mmlab/mmtracking/pull/797
   - [ ] [migration.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/migration.md)
   - [ ] [overview.md](https://github.com/open-mmlab/mmtracking/blob/dev-1.x/docs/en/overview.md)

**Comment by Hiiamein** [comment]
I want to do this and it is a good chance for me to learn how to use mmtracking.

**Reply by l412198735** [reply]
Hao Li +  docs part-3

**Reply by JingweiZhang12** [reply]
> I want to do this and it is a good chance for me to learn how to use mmtracking.

@ArtechStark 
Hi, we'll appreciate it if you can create a PR about the docs part-1 before 12.27. Otherwise, we'll reassign this task to others.

**Comment by DeemoZen** [comment]
Zehua Zeng + docs part-4

**Comment by LihanChen2004** [comment]
Lihan chen + docs part-2

**Reply by JingweiZhang12** [reply]
Please mkdir the `docs/zh_cn` and place the translated docs there. like this: https://github.com/open-mmlab/mmtracking/pull/780/files.
Also, please push the PR into `dev-1.x` branch rather than `master`.
